db/app/app.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const app = express();
+const { getTopics } = require('../controllor/controller');
+
+app.use(express.json());
+app.get('/api/topics', getTopics);
+
+app.use((err, request, response, next) => {
+    console.log(err);
+    response.status(500).send('Server Error!');
+    next(err);
+})
```

Comment:
I like this error handling so far, but you could also add error handling for a 404 status code when an incorrect endpoint request is made with the appropriate testing